package.json

```diff
@@ -44,8 +44,8 @@
   "dependencies": {
     "@slack/logger": "^2.0.0",
     "@slack/oauth": "^1.2.0",
-    "@slack/types": "^1.6.0",
```

Comment:
**Reminder** :: `@slack/types` and `@slack/web-api` need to be updated to appropriate versions prior to merge